common.mk

```diff
@@ -24,7 +24,7 @@ endif
 
 ifeq ($(shell which terraform),)
 $(warning Looks like TerraForm is not installed. This is ok as long as you're not trying to create a new deployment. \
-          Deploying new lambdas is still possible with `make deploy` but `make terraform` will not work.)
+          Deploying new lambdas is still possible with `make package` but `make terraform` will not work.)
```

Comment:
This warning statement is actually not even correct, since `make package` doesn't actually deploy the lambda. Do you think you could fix it?